test/integration/targets/iam_user/aliases

```diff
@@ -0,0 +1,2 @@
+unsupported
```

Comment:
@jillr did the User/Group pieces make it into Terminator yet? (is 'unsupported' needed here)

Reply:
@tremble I don't see anything PRd yet.  For user management we'll also need to make sure there's sufficient guardrails to prevent that terminator and the tests from accessing the actual CI account users though so there will need to be a conversation around how best to do that.  I'd rather let this merge unsupported and enable the tests once we have the CI support in place than block on that.

Reply:
Right now the `iam_user` module is **broken** and can not delete any _real_ AWS user.

It's being almost two months since I open this PR, merge or decline, please.